src/Marvin.Web/Program.cs

```diff
@@ -40,6 +43,15 @@
 
 builder.Services.AddRazorPages();
 
+builder.Services
+    .AddHealthChecksUI()
+    .AddInMemoryStorage();
+/*    .AddPostgreSqlStorage(connectionString);*/
+
+builder.Services.AddHealthChecks()
+    .AddNpgSql(connectionString, name: "Database")
```

Comment:
I guess we will get an error now if we did use the mssql connection but ok if we just go with PostgreSQL

Reply:
Good point and I missed that,  I've now added a database bootstrapper so both SQL Server and Postgres should work